packages/contracts-bedrock/deploy-config/hivenet.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this meant to be empty?

Reply:
Yes, in Hive we swap in a full config json. I thought it was awkward to reference the network in hardhat config / sdk / etc. and not add a placeholder file. So I added this empty thing.